runtime/compiler/ilgen/Walker.cpp

```diff
@@ -1672,6 +1674,36 @@ TR_J9ByteCodeIlGenerator::stashArgumentsForOSR(TR_J9ByteCode byteCode)
    TR::MethodSymbol *symbol = symRef->getSymbol()->castToMethodSymbol();
    int32_t numArgs = symbol->getMethod()->numberOfExplicitParameters() + (symbol->isStatic() ? 0 : 1);
 
+#if defined(J9VM_OPT_OPENJDK_METHODHANDLE)
+   // invokehandle/invokedynamic bytecode implicitly push an object from side
+   // table as the last argument of the call. This is part of the behavior of
+   // the bytecode in Openjdk MethodHandle implementation. The object should
+   // not be stashed.
+   // The unresolved case will have one more implicit argument pushed onto the stack,
+   // as the MemberName from side table is unknown, the target method is not known at
+   // compile time, thus, the JIT uses MethodHandle.linkToStatic to represent the call.
+   // MethodHandle.linkToStatic expects the last argument to be MemberName object
```

Comment:
Can you also make it clear in the comment here that the MemberName object isn't being stashed either.  At least that's what I think is happening based on the code below.